<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TeaWindow</class>
 <widget class="QMainWindow" name="TeaWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>665</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>教师用户界面</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>课程查看</string>
       </attribute>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>91</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>您教授的课程：</string>
        </property>
       </widget>
       <widget class="QTableView" name="tableView_c">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>50</y>
          <width>431</width>
          <height>192</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>学生信息</string>
       </attribute>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>54</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>学生信息：</string>
        </property>
       </widget>
       <widget class="QTableView" name="tableView_s">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>51</y>
          <width>491</width>
          <height>181</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>成绩录入</string>
       </attribute>
       <widget class="QTableView" name="tableView_g">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>10</y>
          <width>491</width>
          <height>251</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_submit">
        <property name="geometry">
         <rect>
          <x>550</x>
          <y>150</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>上传</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_revert">
        <property name="geometry">
         <rect>
          <x>550</x>
          <y>200</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>543</x>
          <y>30</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>当前所选科目：</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_g">
        <property name="geometry">
         <rect>
          <x>540</x>
          <y>70</y>
          <width>91</width>
          <height>23</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>修改密码</string>
       </attribute>
       <widget class="QFrame" name="frame">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>30</y>
          <width>271</width>
          <height>191</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>22</y>
           <width>91</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>输入当前密码</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>63</y>
           <width>91</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>输入新密码</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>104</y>
           <width>91</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>再次输入新密码</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButtonpwd">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>140</y>
           <width>80</width>
           <height>24</height>
          </rect>
         </property>
         <property name="text">
          <string>确认修改</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEditpwd">
         <property name="geometry">
          <rect>
           <x>108</x>
           <y>22</y>
           <width>151</width>
           <height>23</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_pwdn">
         <property name="geometry">
          <rect>
           <x>108</x>
           <y>63</y>
           <width>151</width>
           <height>23</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_pwdn2">
         <property name="geometry">
          <rect>
           <x>108</x>
           <y>104</y>
           <width>151</width>
           <height>23</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>665</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
